docs/admin/projects.rst

```diff
@@ -764,30 +763,29 @@ By default the component is visible to anybody who has access to the project,
 even if the person can not perform any changes in the component. This makes it
 easier to keep translation consistency within the project.
 
-Enable this in case you want to grant access to this component explicitly -
-the project level permissions will not apply and you will have to specify
-component or component list level permission in order to grant access.
+Turning this on at component-, or component-list level takes over
```

Comment:
```suggestion
Turning this on a component-, or component-list level takes over
```
or
```suggestion
Turning this at the component-, or component-list level takes over
```

Reply:
turning this on for components or projects.

I am not exactly sure what "projects" are, if not just lists of projects?
Does this also mean specific component lists linked to groups that users can be put in?

Reply:
Haven't noticed it before, but your snippet is much more confusing that the original... 

This describes a boolean setting in the component.

If it's turned off (default) the project-level access control settings are used.
If it's turned on the access is restricted by default and you have to define (or used one of predefined groups for `custom` or `private` access control) a group with component-level or component-list-level access control as permissions for the project-level access control settings are not used.

I'd suggested to refere the access control doc here, but as you know those are in quite badly shape yet...

Reply:
@Fat-Zer 
"Turning it _on_" denotes both that it is normally off, and a boolean on/off setting(?).
How about "If you turn this on, at either ___ "?

Reply:
The problem with the wording here is, it just doesn't make sense to "*turn this setting on at some level*". It's just a setting for a specific component, isn't it? You just "*turn this on for a specific component*" and then you "*manage access to the component on a component- or component-list-level*" due to it overrides the project-level access control...

As for now I like the original variant more...

Reply:
Anyway, what's the problem with the original? I might try to help with a rephrasing it, but I just don't see why bother... At most I would replace a dash (`-`) with a colon (`:`) in it (thou even about that I'm not 100% sure)...

Reply:
please don’t leave your conversations open and solution-less